<?xml version="1.0"?>
<!--***********************************************************
	MoveIt Planner Executor Server Node
	Emily Sheetz, NSTGRO VTE 2023

	run:
		$ roslaunch val_moveit_planner_executor moveit_planner_executor_server_node.launch
************************************************************-->
<launch>
	
	<arg name="debug" default="false"/>
	<arg unless="$(arg debug)" name="launch_prefix" value="" />
	<arg if="$(arg debug)" name="launch_prefix" value="xterm -e gdb **args" />

	<!-- parameters for specifying IHMCMsgInterface topic -->
	<arg name="ihmc_msg_interface_moveit_topic" default="/IHMCInterfaceNode/moveit_planned_robot_trajectory"/>
	<arg name="ihmc_msg_interface_receive_moveit_traj_topic" default="/IHMCInterfaceNode/receive_moveit_trajectories"/>

	<node launch-prefix="$(arg launch_prefix)" pkg="val_moveit_planner_executor" type="moveit_planner_executor_server_node" name="ValkyrieMoveItPlannerExecutorServerNode" output="screen">
		<param name="ihmc_msg_interface_moveit_topic" type="string" value="$(arg ihmc_msg_interface_moveit_topic)"/>
		<param name="ihmc_msg_interface_receive_moveit_traj_topic" type="string" value="$(arg ihmc_msg_interface_receive_moveit_traj_topic)"/>
		<!--<param name="" type="" value=""/> -->
	</node>

</launch>
